examples/ola-rdm.cpp

```diff
@@ -315,7 +317,7 @@ void RDMController::HandleResponse(const ola::client::Result &result,
     } else {
       memcpy(reinterpret_cast<uint8_t*>(&backoff_time), response->ParamData(),
              sizeof(backoff_time));
-      unsigned int timeout = 100 * backoff_time;
+      unsigned int timeout = 100 * NetworkToHost(backoff_time);
```

Comment:
This is feeling more and more to me like we should be providing a method as part of RDMResponse to do most of the leg work for you; it's a lot of potential for bugs for someone to implement in their own client.